services/app-web/src/common-code/healthPlanFormDataType/healthPlanFormData.ts

```diff
@@ -163,10 +162,14 @@ function programNames(
 
 function packageName(
     submission: HealthPlanFormDataType,
-    statePrograms: ProgramArgType[]
+    statePrograms: ProgramArgType[],
+    programIDs?: string[]
 ): string {
     const padNumber = submission.stateNumber.toString().padStart(4, '0')
-    const pNames = programNames(statePrograms, submission.programIDs)
+    const pNames =
+        programIDs && programIDs.length > 0
+            ? programNames(statePrograms, programIDs)
```

Comment:
Can you add a comment in here about why the ternary is needed? I think we could easily forget about this context and look at this function later confused how `programIds` could not exist